{"body": "\nHi, just another question.\n\n[<img class=\"align-full\" src=\"http://api.ning.com:80/files/*ix1zXEQDKOjs65G7XlxXti-*MmDje7NxdQj7N668z-FtXFGsfOfOAho*vVItsok93lsE2gQuD5vZNf0GJXdMOld8Ogpf2Cz/gendaylit.png?width=721\" width=\"721\"/>](http://api.ning.com:80/files/*ix1zXEQDKOjs65G7XlxXti-*MmDje7NxdQj7N668z-FtXFGsfOfOAho*vVItsok93lsE2gQuD5vZNf0GJXdMOld8Ogpf2Cz/gendaylit.png)I still cannot seem to run my simulation, because of this error with gendaylit. I have checked my folder and I see gendaylit in my Radiance bin. May I know if there is any reason for this?\n\n[<img class=\"align-full\" src=\"http://api.ning.com:80/files/*ix1zXEQDKNgXgr*jBfUUXJ03CHgZcNVGD9TVnghwCU2*xgEViyAmliD87SkoYeqfEk64QXMJ1yccxoOHP8RAKGU2DJQN8mg/gendaylit.png?width=721\" width=\"721\"/>](http://api.ning.com:80/files/*ix1zXEQDKNgXgr*jBfUUXJ03CHgZcNVGD9TVnghwCU2*xgEViyAmliD87SkoYeqfEk64QXMJ1yccxoOHP8RAKGU2DJQN8mg/gendaylit.png)\n\n\n\nRegards, Nicole.\n", "attachments": [], "created_by_name": "Nicole Chung", "created_at": "February 4, 2017 at 8:49pm", "created_by": "NicoleChung", "parent_id": "reply_2985220_Comment_1685268", "id": "reply_2985220_Comment_1685348"}